docs/upcoming_changes/9249.np_support.rst

```diff
@@ -0,0 +1,3 @@
+Adds support for checking if dtypes are equal.
+========================================================================================================================================================================
```

Comment:
I think this "underline" needs to be the same length as the title line?

Reply:
That sounds right, fixed!